tencent-cloud/mysql/create.go

```diff
@@ -33,25 +35,63 @@ func NewCredential() (*common.Credential, *profile.ClientProfile) {
 	return credential, cpf
 }
 
-func (tm TcloudMySQL) Create() {
+func (tm TcloudMySQL) Create() (bool, error) {
+
+	// Init client
 	credential, cpf := NewCredential()
-	client, _ := cdb.NewClient(credential, tm.Region, cpf)
+	client, err := cdb.NewClient(credential, tm.Region, cpf)
+	if err != nil {
+		return false, err
+	}
 
+	// Check if instance with the same name already exists
+	exists, err := checkInstanceExists(client, tm.InstanceName)
+	if err != nil || exists {
+		return exists, errors.Wrap(err, "failed to check instances")
```

Comment:
this is kind of a weird error handling. wouldn't this always hit the caller's `err != nil` block and not the `exists` block?

Reply:
oh, errors.Wrap() probably doesn't return an error if the input error is nil?

if thats the case i guess its ok, but this handler could be written more clearly / less tricky